bangazonapi/views/profile.py

```diff
@@ -178,15 +178,15 @@ def cart(self, request):
             try:
                 open_order = Order.objects.get(
                     customer=current_user, payment_type=None)
-                line_items = OrderProduct.objects.filter(order=open_order)
-                line_items = LineItemSerializer(
-                    line_items, many=True, context={'request': request})
+                lineitems = OrderProduct.objects.filter(order=open_order)
+                lineitems = LineItemSerializer(
+                    lineitems, many=True, context={'request': request})
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False, context={
                                                 'request': request}).data
-                cart["order"]["line_items"] = line_items.data
-                cart["order"]["size"] = len(line_items.data)
+                cart["order"]["lineitems"] = lineitems.data
```

Comment:
There's still a duplicate in the response either this line should be deleted or the OrderSerializer should be updated

Reply:
I am not seeing this in my local branch, i'm also only getting one 'lineitems' via PostMan.

Reply:
That's because `lineitems`matches what is in the serializer. It's overwriting what's already there